Xwt/Xwt/TreeStore.cs

```diff
@@ -69,6 +69,11 @@ public TreeStore (params IDataField[] fields)
 		ITreeStoreBackend Backend {
 			get { return (ITreeStoreBackend)BackendHost.Backend; }
 		}
+
+		public void SetSortField (IDataField field, ColumnSortDirection order)
```

Comment:
As far as I remember, for the Gtk+ backend this is (was) not possible, because the Xwt column does not have access to the Gtk+ tree store, which is where sorting is controlled in that toolkit. I remember there was some Xwt api concern about this PR which is the reason for it not being merged, but the discussion doesn't seem to have happened here, perhaps it was in #mono or #monodevelop.